docs/development_suite/overview-dev-suite.md

```diff
@@ -35,3 +54,5 @@ Mia-Platform Console is organized into hierarchical levels to help users navigat
 - **Project Sections:** Designated areas within a project that delineate and organize functionalities and tasks.
 
 This structure ensures clear and scalable management of resources and activities within Mia-Platform Console.
+
+Discover all the **features and capabilities** of Mia-Platform Console by exploring the menu items on the left.
```

Comment:
In addition to suggesting the menu, I would also cross-link to the specific overviews of Console, Fast Data, etc

Reply:
My proposal:
Discover all the **features and capabilities** of Mia-Platform Console, by exploring the menu items on the left of this page, and the other products [Mia-Platform Fast Data](https://docs.mia-platform.eu/docs/fast_data/what_is_fast_data) and [Mia-Platform Marketplace](https://docs.mia-platform.eu/docs/marketplace/overview_marketplace).

Reply:
add also Microfrontend Composer